packages/providers/src/core/PoapMomentsApi/PoapMomentsApi.ts

```diff
@@ -164,6 +165,22 @@ export class PoapMomentsApi implements MomentsApiProvider {
     }
   }
 
+  /**
+   * Update a moment using the provided input
+   * @param {string} id - The moment id
+   * @param {PatchMomentInput} input - The input for updating a moment
+   */
+  public async patchMoment(id: string, input: PatchMomentInput): Promise<void> {
```

Comment:
What errors can this endpoint generate?

Reply:
ATM 404 moment not found and BadPatchRequest when cid is invalid but we are not doing anything with those errors. We can see of adding a good error handling strategy when this becomes an issue if we are ok